Source/player.cpp

```diff
@@ -1909,7 +1906,8 @@ StartPlayerKill(int pnum, int earflag)
 						i = NUM_INVLOC;
 						while (i--) {
 							pdd = (i + player._pdir) & 7;
-							PlrDeadItem(player, pi, offset_x[pdd], offset_y[pdd]);
+							Point offset = Point { 0, 0 } + static_cast<Direction>(pdd);
+							PlrDeadItem(player, pi, offset.x, offset.y);

```

Comment:
In case this isn't obvious this is the same as calling `right[pdir]` `i` times. I wonder if the is a cleaner way we could write this part.

Reply:
I'll revisit these in a subsequent PR where we go through every function in the player file.